.github/workflows/lint.yml

```diff
@@ -27,6 +27,15 @@ jobs:
         run: |
           python .github/scripts/check_index_rst.py docs/en/index.rst
           python .github/scripts/check_index_rst.py docs/zh_cn/index.rst
+      - name: Check markdown link
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          use-verbose-mode: 'yes'
+#          check-modified-files-only: 'yes'
+          config-file: '.github/md-link-config.json'
+          folder-path: 'docs/en, docs/zh_cn'
```

Comment:
Does the tool only work for markdown links or beyond?

Reply:
I saw that even some wrong links in python files got detected.

Reply:
It only checks the links in markdown.